diff -Naur iomod_isis3/build_3.4.sh iomod_isis3_new/build_3.4.sh
--- iomod_isis3/build_3.4.sh	2016-10-07 10:03:08.784250446 -0700
+++ iomod_isis3_new/build_3.4.sh	2016-10-07 10:08:38.587609100 -0700
@@ -1,17 +1,20 @@
 #! /bin/sh
 echo Compiling ISIS3 module.... 3.4 version
 ISIS3DIR=/usgs/pkgs/isis3/isis
-QTSRCDIR=/usr
+QTSRCDIR=/usr/include/qt5
 ARCH=64
-LD_RUN_PATH=$ISIS3DIR/3rdParty/lib
-echo $LD_RUN_PATH
-g++ -DHAVE_CONFIG_H -m$ARCH -fPIC -g -I.. -I../libltdl -I$ISIS3DIR/inc -I$QTSRCDIR/include -I$QTSRCDIR/include/QtCore -Wall iomod_isis3.4.c -c -o iomod_isis34.o
+LD_RUN_3PTY=$ISIS3DIR/3rdParty/lib
+LD_RUN_ISIS=$ISIS3DIR/lib
+
+echo $LD_RUN_3PTY $LD_RUN_ISIS
+
+g++ -DHAVE_CONFIG_H -m$ARCH -fPIC -g -I.. -I../libltdl -I$ISIS3DIR/inc -I$QTSRCDIR -I$QTSRCDIR/QtCore -Wall iomod_isis3.4.c -c -o iomod_isis34.o
 if [ $? != 0 ]; then
     echo Compile failed.
     exit 1
 fi
 echo Linking ISIS3 module...
-g++ -m$ARCH -fPIC -shared -Wl,-soname,isis34.dvio -Wl,-rpath=$ISIS3DIR/3rdParty/lib -o isis34.dvio iomod_isis34.o  -L$ISIS3DIR/lib -L$ISIS3DIR/3rdParty/lib -lisis3 -lkdu_a63R -lgsl -lgeos_c -lqwt -lxerces-c -lprotobuf -lQtDBus
+g++ -m$ARCH -fPIC -shared -Wl,-soname,isis34.dvio -Wl,-rpath=$LD_RUN_3PTY -Wl,-rpath=$LD_RUN_ISIS -o isis34.dvio iomod_isis34.o  -L$LD_RUN_ISIS -L$LD_RUN_3PTY -lisis3 -lkdu_a63R -lgsl -lgeos_c -lqwt -lxerces-c -lprotobuf -lQtDBus
 if [ $? != 0 ]; then
     echo Link failed.
     exit 1
diff -Naur iomod_isis3/iomod_isis3.4.c iomod_isis3_new/iomod_isis3.4.c
--- iomod_isis3/iomod_isis3.4.c	2016-10-07 10:03:08.788250438 -0700
+++ iomod_isis3_new/iomod_isis3.4.c	2016-10-07 09:09:38.792483319 -0700
@@ -35,7 +35,7 @@
 
 #define string_equal(a,b) !strcmp(a,b)
 #define string_equal_ic(a,b) !strcasecmp(a,b)
-#define KWD_TO_CHAR_PTR(x) (char *)((x).toAscii().data())
+#define KWD_TO_CHAR_PTR(x) (char *)((x).toLatin1().data())
 
 #define CUBE_ASSOC "cube_association"
 #define SAMPLE_STR "Sample"
